resources/views/layout/welcome.blade.php

```diff
@@ -19,8 +19,10 @@
 
 
 @include('partials.nav-partials')
-
+@if(!isset($user))
 @yield('content')
+@endif
+@yield('dash')
```

Comment:
Only design conditions are allowed in view templates, not routing logic. This will be handled in the controller. Leave only `@yield('content')` here.